extensions/amp-apester-media/0.1/monetization/companion/bottomAd.js

```diff
@@ -0,0 +1,74 @@
+/**
+* Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS-IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import {Services} from '../../../../../src/services';
+import {createElementWithAttributes} from '../../../../../src/dom';
+import {dict, getValueForExpr} from '../../../../../src/core/types/object';
+const ALLOWED_AD_PROVIDER = 'gpt';
+
+/**
+ * @param {!JsonObject} media
+ * @param {!AmpElement} apesterElement
+ */
+export function handleCompanionBottomAd(media, apesterElement) {
+  const bottomAdOptions = getValueForExpr(
+    /**@type {!JsonObject}*/ (media),
+    'campaignData.bottomAdOptions'
+  );
+  if (!bottomAdOptions) {
+    return;
+  }
+  const enabledBottomAd = getValueForExpr(
+    /**@type {!JsonObject}*/ (bottomAdOptions),
+    'enabled'
+  );
+  if (
+    enabledBottomAd &&
+    bottomAdOptions['videoPlayer'] === ALLOWED_AD_PROVIDER
+  ) {
+    const slot = bottomAdOptions['tag'];
+    const bannerSizes = [[300, 50]];
```

Comment:
Is there a reason behind these preset values being chosen? I see `50` corresponds to the hardcoded bottom ad height -- why `300` for width?

Reply:
At this moment we use only 300x50 size for bottom ad blocks as a most popular for that placement.